tools/Resources.py

```diff
@@ -86,3 +88,32 @@ def get_latest_stat(self, target, token, uuid, key):
         except HTTPError as e:
             raise HTTPError(
                 "Request failed for statkey: " + key + " and target: " + target + "\nerror message:" + str(e))
+
+    def get_latest_stat_multiple(target, token, uuids, key):
+
+        url = "https://" + target + "/suite-api/api/resources/stats/latest/query"
+        headers = {
+            'Content-Type': "application/json",
+            'Accept': "application/json",
+            'Authorization': "vRealizeOpsToken " + token
+        }
+        payload = {
+                "resourceId": uuids,
+                "statKey": [key]
+        }
+        disable_warnings(exceptions.InsecureRequestWarning)
+        try:
+            response = requests.post(url,
+                                     data=json.dumps(payload),
+                                     verify=False,
+                                     headers=headers)
+        except Exception as e:
+            print("Problem getting stats Error: " + str(e))
+            return False
+
+        if response.status_code == 200:
+            return response.json()['values']
+        else:
+            print("Return code not 200 for " + str(key) + ": " + response.json())
```

Comment:
Nice error handling. But we have to do it the same for all methods. I find the 200 as return code not clear enough as an error message. It would be better to know from which method the request just comes. We have several methods that make requests. What do you think?

Reply:
What do you suppose instead? I don't want do have the whole traceback in here, the originating caller however would be interesting indeed.

Reply:
Ok, that's right. But what does not work is to concatenate string and dict at `response.json()`.